src/pages/Settings/Password.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+	<div class="row">
+		<div class="column small-12 large-8">
```

Comment:
I believe this class should be "columns"

Reply:
You can use either column or columns, since there is only 1, in this case 'column' makes grammatical sense